### PR TITLE
UI fixes and enhancements

### PR DESCRIPTION
**Pull request by Subhadiptech**
UI fixes:
- The third text was not responsive as it had some horizontal bias of 0.0. I fixed it by removing the bias and adding horizontal constraints.

UI enhancements:I
- The line between Study Less and remember more has been added with a gradient to make it look better.

### CODE REVIEW

**Comment by lukstbit** [comment]
Thank you for the contribution. 
I haven't checked the changes but does your code take in consideration the different themes we offer to the user(asking this because you added a hard coded color in the layouts)? Also we ask for PRs with changes to the UI that they provide some images showing the UI changes.

**Comment by Subhadiptech** [comment]
I got it now, I will apply theme styles instead of hard code. Thanks for pointing it out.

**Comment by Subhadiptech** [comment]
I am sharing the image of the changes made

**Comment by Subhadiptech** [comment]
Before :

Screenshot 2022-10-25 at 11 54 17 AM

After:

Screenshot 2022-10-25 at 11 53 11 AM

File: AnkiDroid/src/main/res/drawable/gradient_line_horizontal.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android">
+    <gradient
+        android:centerColor="#dddddd"
```
Comment (lukstbit) [inline AnkiDroid/src/main/res/drawable/gradient_line_horizontal.xml:4]: Extract this color to the end of colors.xml and name it as `intro_center_separator`

File: AnkiDroid/src/main/res/drawable/gradient_line_horizontal.xml [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+        android:endColor="@color/white"
+        android:startColor="@color/white" />
```
Comment (lukstbit) [inline AnkiDroid/src/main/res/drawable/gradient_line_horizontal.xml:6]: In order to handle the different themes you could replace this hardcodded color with the attribute from the theme responsible for the background color: 

`?android:colorBackground`

File: AnkiDroid/src/main/res/layout/introduction_layout.xml [hunk 3]
```diff
@@ -58,9 +58,9 @@
 
     <View
         android:id="@+id/title_divider"
-        android:layout_width="100dp"
+        android:layout_width="160dp"
```
Comment (lukstbit) [inline AnkiDroid/src/main/res/layout/introduction_layout.xml:61]: There is no need to make this wider, leave it as it is.

File: AnkiDroid/src/main/res/layout/introduction_layout.xml [hunk 4]
```diff
@@ -58,9 +58,9 @@
...
         android:layout_height="2dp"
-        android:background="#dddddd"
+        android:background="@drawable/gradient_line_horizontal"
```
Comment (lukstbit) [inline AnkiDroid/src/main/res/layout/introduction_layout.xml:63]: Rename the drawable file to `horizontal_gradient_separator`.

File: AnkiDroid/src/main/res/layout/introduction_layout.xml (hunk 3 above)
Comment (Subhadiptech) [reply AnkiDroid/src/main/res/layout/introduction_layout.xml:61]: As the line was previously a normal line, it was looking fine with 100dp, but as I added gradient and the start and end color are white, it required some more length for better look. Please Correct me if I was wrong. Thank you

**Comment by Subhadiptech** [comment]
I am making the required changes. Thanks for the feedbacks @lukstbit

**Comment by Subhadiptech** [comment]
I have made the required changed @lukstbit

File: AnkiDroid/src/main/res/layout/homescreen.xml [hunk 5]
```diff
@@ -1,7 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <androidx.coordinatorlayout.widget.CoordinatorLayout xmlns:android="http://schemas.android.com/apk/res/android"
-                                           android:layout_width="match_parent"
-                                           android:layout_height="match_parent"
-                                           >
-    <include layout="@layout/deck_picker"/>
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
```
Comment (david-allison) [inline AnkiDroid/src/main/res/layout/homescreen.xml:5]: This seems like a no-op, could it be reverted?

**Review by david-allison — CHANGES_REQUESTED** [review]
LGTM. One nitpick

**Review by lukstbit — APPROVED** [review]
Thanks!

File: AnkiDroid/src/main/res/layout/homescreen.xml (hunk 5 above)
Comment (Subhadiptech) [reply AnkiDroid/src/main/res/layout/homescreen.xml:5]: It was not changed by me @david-allison

Comment (Subhadiptech) [reply AnkiDroid/src/main/res/layout/homescreen.xml:5]: > The formatting right? @david-allison

Comment (lukstbit) [reply AnkiDroid/src/main/res/layout/homescreen.xml:5]: @Subhadiptech  Yes, the change is unrelated to the other changes.

Comment (Subhadiptech) [reply AnkiDroid/src/main/res/layout/homescreen.xml:5]: Sure, I will revert

**Comment by Subhadiptech** [comment]
I have reverted the un wanted changes. @david-allison  @lukstbit

**Comment by Subhadiptech** [comment]
> LGTM. One nitpick

It's updated

**Review by mikehardy — APPROVED** [review]
All comments appear to be handled

**Comment by mikehardy** [comment]
First PR merged! Congratulations :smile:

**Comment by Subhadiptech** [comment]
Thanks 🙏

**Pull request merged**